spec/config/dPathRegex/dPathRegex.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { dPathRegex, whitespaceDPathRegex } from 'config';
+import { nospaceValid, nospaceInvalid, whitespaceValid, whitespaceInvalid } from './testStrings';
+
+describe('dPath regex', () => {
+  it('should match valid strings', () => {
+    nospaceValid.forEach(path => {
+      expect(dPathRegex.test(path)).toBeTruthy();
+    });
+  });
+
+  it("shouldn't match invalid strings", () => {
+    nospaceInvalid.forEach(path => {
+      expect(dPathRegex.test(path)).toBeFalsy();
+    });
+  });
+});
+
+describe('whitespace dPath regex', () => {
+  it('should', () => {
+    whitespaceValid.forEach(path => {
+      expect(whitespaceDPathRegex.test(path)).toBeTruthy();
+    });
+  });
+
+  it("shouldn't match invalid strings", () => {
+    whitespaceInvalid.forEach(path => {
+      expect(whitespaceDPathRegex.test(path)).toBeFalsy();
+    });
+  });
+});
```

Comment:
I think more valuable than testing the regex's, we should be testing our validator function. We don't use the regexes in code directly anywhere, they're all hidden behind the one validator function.

With that in mind, this should probably expand on the existing tests in /spec/libs/validators.spec.ts

Reply:
Yeah now that I'm looking at this more, there's actually a false positive here in having a test for the whitespace regex thinking that that would work for user input, but the isValidPath function only uses the other regex. So those would be invalid in the app.